.github/workflows/build.yml

```
@@ -45,32 +44,39 @@ jobs:
       with:
           submodules: 'recursive'
           fetch-depth: 1
 
 
+      # The Bash version installed on macOS is ancient. We need a newer version
+    - name: Install Bash
+      if: startswith(matrix.os, 'macos')
+      run: brew install bash
+
     - name: Set environment variables
       run: $GITHUB_WORKSPACE/.github/workflows/set_environment_variables.sh "${{ matrix.os }}" "${{ matrix.type }}" "${{ matrix.arch }}" "${{ runner.workspace }}"
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2086:info:1:1: Double quote to prevent globbing and word splitting

(shellcheck)